### PR TITLE
feat(.goreleaser.yml): add sha256 check to release notes

### PR DESCRIPTION
**Pull request by arthurzenika**
Improve commands so that the checksum of the downloaded file is checked before cosign is checked.

This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

**Review by sudermanjr — APPROVED** [review]
Thanks!

**Comment by arthurzenika** [comment]
While I'm at it (and maybe I should describe this in an issue), it would be even nicer if the generated notes would include the version, for example for the last release : 

```
sha256sum -c goldilocks_4.7.2_checksums.txt --ignore-missing
cosign verify-blob goldilocks_4.7.2_checksums.txt --signature=goldilocks_4.7.2_checksums.txt.sig  --key https://artifacts.fairwinds.com/cosign.pub
```

not familiar with gorelease features though

**Comment by sudermanjr** [comment]
You should be able to use the template placeholder `{{.Version}}` in the release note footer, looking here: https://goreleaser.com/customization/release/?h=release

File: .goreleaser.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ release:
     You can verify the signatures of both the checksums.txt file and the published docker images using [cosign](https://github.com/sigstore/cosign).
 
     ```
+    sha256sum -c checksums.txt --ignore-missing
```
Comment (sudermanjr) [inline .goreleaser.yml:35]:
```suggestion
    sha256sum -c goldilocks_v{{ .Major }}.{{ .Minor }}.{{ .Patch }}_checksums.txt --ignore-missing
```

File: .goreleaser.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ release:
...
     cosign verify-blob checksums.txt --signature=checksums.txt.sig  --key https://artifacts.fairwinds.com/cosign.pub
```
Comment (sudermanjr) [inline .goreleaser.yml:36]:
```suggestion
    cosign verify-blob goldilocks_v{{ .Major }}.{{ .Minor }}.{{ .Patch }}_checksums.txt --signature=goldilocks_v{{ .Major }}.{{ .Minor }}.{{ .Patch }}_checksums.txt.sig  --key https://artifacts.fairwinds.com/cosign.pub
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]

**Review by sudermanjr — APPROVED** [review]
Thanks!

**Comment by sudermanjr** [comment]
Latest release notes are much nicer. Thanks again!

**Pull request merged**